code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -38,19 +49,16 @@
 	var/obj/item/slimepotion/slime/current_potion
 	var/obj/machinery/monkey_recycler/connected_recycler
 
-	icon_screen = "slime_comp"
-	icon_keyboard = "rd_key"
-
 /obj/machinery/computer/camera_advanced/xenobio/Initialize(mapload)
-	..()
-	return INITIALIZE_HINT_LATELOAD
+	. = ..()
+	locate_recycler()
```

Comment:
From what I can tell, this can go wrong depending on the map layout. If the monkey recycler is initialised before the camera then it goes correctly. Otherwise, if the camera goes first then the `monkey_recyclers` list is empty.
It's best to keep late init here for this reason.

It works because the `GrantActions` proc locates the recycler when it's not correctly loaded. Not the best place to fix a missing reference though